### ISSUE TITLE
Documentation has error ; external oscillator can't be working with DIP switch indications.

### ISSUE THREAD

**Issue by Boris-Edu** [Xinyuan-LilyGO/ESP32_S2#2]
Hello,

TLDR :
Documentations on the dip switch for the external oscillator are wrong and should be inversed.

I wasn't managing to get the external crystal to oscillate on my T8 board.
It's a Lilygo ESP32_S2 T8 (20-6-18 rev 1.1).

Indeed I made sure to enable the external 32khz crystal in config. 
In SDKCONFIG I have : CONFIG_RTC_CLK_SRC_EXT_OSC=y

I measured directly on the oscillator, as the oscillator gets disconnected from the circuit from the pinout when the dipswitch is configured.
I wasn't always getting the ""32 kHz XTAL not found, switching to internal 90 kHz oscillator" message on startup, but I never measured the expected waveform at the pins.

I checked the T9 datasheet and observed the indications on the Lilygo website weren't matching what's described on the schematic.
![Screenshot 2022-08-01 at 01 45 30](https://user-images.githubusercontent.com/37216326/182050366-0f1147fc-8a84-4fd9-9aa2-976d8866cecb.jpg)
![Screenshot 2022-08-01 at 01 45 06](https://user-images.githubusercontent.com/37216326/182050367-94732dac-a346-4539-8fc4-88a34c9c09db.jpg)

I orientated the schematic so it matches the picture.
You want switch 1 and 3 to be pushed up to have access to the oscillator (indicated by XP/XN).

I confirmed by measuring the voltage at the pins, and proceeded to set the DIP switch to the GPIO settings.
And, what a surprise, I now get a signal from the oscillator : 

![Screenshot 2022-08-01 at 01 42 12](https://user-images.githubusercontent.com/37216326/182050403-a29f56b6-9234-499b-ba85-6410630fa5b9.jpg)
Sorry for the lack of proper probing, I'm not in my lab, did the screen capture with my elbow, lol.

I was surprised by the signal's low voltage, but it looks that it's what's expected, voltages are close to what's captured on this other issue (see capture for 32k on branch 4.2) : 
https://github.com/espressif/esp-idf/issues/6621

Sincerely,
